src/scenes/DeliverablesRouter/DeliverablePageMessage.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Box, useTheme } from '@mui/material';
+import { Button, Message } from '@terraware/web-components';
+import { ViewProps } from 'src/components/DeliverableView/types';
+import { useLocalization } from 'src/providers/hooks';
+import strings from 'src/strings';
+
+const DeliverableView = (props: ViewProps): JSX.Element => {
```

Comment:
It might be ok to have 2 separate implementations, one for the participant view and one for the accelerator console view. Or move this under src/components. Two separate implementations might be ok.
I have posed a question on what to do for edit reason. We have a reject reason flow, maybe we can reuse that in the accelerator console view's implementation (change the title of the modal to 'Edit Reason' and button name to 'Save' or something like that). Thoughts?

Reply:
I could see the case for either approach, but two implementations sounds reasonable to me. I'm not sure about the reject reason flow, but only because I'm not familiar with it and how much we'd get/save by using it.

Reply:
Thanks, Josh. Yes two implementations might be cleaner for now.
Reusing the reject/reason flow but changing the modal title and button text will get you an 'edit reason' feature. It will show a text area to accept new 'reason' (soon to be 'feedback') and updates the deliverable.
That's totally fine if we do this as a separate task, I can take it since I added the original flow. Also, waiting on Chudo to respond in figma on how he sees the edit reason flow.

Reply:
Are we still waiting on other feedback related to this ticket?

Reply:
@joshbuchea looks like the accelerator console version of the page message is missing now.

Reply:
@karthikbtf please clarify exactly which changes you expect to see so I know I'm making the correct changes 🙏

Reply:
No worries, my mistake for adding confusion. I was referring to two separate views instead of a shared view. The separate views were being referred to as implementations. We can do this in a separate PR that's totally fine.

Reply:
@joshbuchea we could capture the other effort as a separate JIRA.

Reply:
I'll create a separate ticket.